pkg/tests/api/alerting/api_alertmanager_test.go

```diff
@@ -1075,7 +1075,6 @@ func TestAlertRuleCRUD(t *testing.T) {
 				require.NoError(t, err)
 
 				assert.Equal(t, res.Message, tc.expectedMessage)
-				assert.NotEmpty(t, res.TraceID)
```

Comment:
why the `TraceID` no more required?

Reply:
Due to below and api tests doesn't have tracing enabled.

> In addition, don't attach `traceID` to context when it's empty and by that not including `traceID=00000000000000000000000000000000` in logs.